code/form/TimeZoneField.php

```diff
@@ -25,12 +25,18 @@ public function __construct($name, $title = null, $source = null, $value = '', $
 		// if a source is given it should be a SS_List of TimeZone objects
 		if (!is_null($source) && !is_a($source, "SS_List")) {
 			throw new Exception("\$source must be null to use the provided values, or an SS_List of TimeZone objects");
+			// why return false? when @return void
 			return false;
 		}
 
 		// if no source has been defined we assume the default time zones will be used
 		if (!$source) $source = TimeZone::get();
 
+		// default to setting defined in php configuration
+		if ($defaultTZ = TimeZone::get()->where("Identifier='".date_default_timezone_get()."'")->First()) {
```

Comment:
to be honest, i don't know the difference. However, I did initially use the filter method but I found an issue and the same issue also happened with I used where(). I prefer where() because thats how Rails does it. Turns out, the following code fails to run the query (hint: double quotes).

DataObject::get_one('TimeZone', array('Identifier' =>  date_default_timezone_get());